locator/tablets.hh

```diff
@@ -228,6 +228,26 @@ struct resize_decision {
     sstring resize_name() const;
 };
 
+struct table_load_stats {
+    size_t size_in_bytes = 0;
+    // Stores the minimum phase among all replicas, as coordinator wants to know if
+    // all replicas are split ready for the current phase.
+    api::timestamp_type split_ready_phase{};
+
+    table_load_stats& operator+=(const table_load_stats& s) noexcept;
+    friend table_load_stats operator+(table_load_stats a, const table_load_stats& b) {
+        return a += b;
+    }
+};
+
+struct load_stats {
+    std::unordered_map<table_id, table_load_stats> tables;
+
```

Comment:
We could add total disk space and free disk space, but let's leave that for later until we know more. These stats could help when moving between instances with different density.

Reply:
Makes sense.